src/layout/section/landingGridSection/LandingGridSection.style.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+
+import { color, responsiveBreakpoints } from '../../../_utils/branding'
+
+const Grid = styled.div`
+  display: grid;
+  height: 100vh; //@todo: strech
```

Comment:
We have many issue with 100vh on mobile, we might want to avoid this value.

Reply:
I 've moved 100vh to a media query big screens I hope would help and be able to use it